package-lock.json

```diff
@@ -52,7 +52,7 @@
         "node": ">=14"
       },
       "peerDependencies": {
-        "puppeteer": "16.1.0"
+        "puppeteer": "^15.3.1 || ^16.1.0"
```

Comment:
I think the old version range still mentioned here

Reply:
Oh, :facepalm: . Thanks for spotting this! I've amended the change to the last commit.